### ISSUE TITLE
Form rendering displays nothing if HTML is not enabled in default template extensions

### ISSUE THREAD

**Issue by anita-chouhan** [verbb/formie#841]
**Description**
I tried to render a new form in the Twig template and found that it is not showing. After debugging a while I found that it is required to add `html` in `defaultTemplateExtensions` in `general.php` file to show Formie forms in Twig.

**Steps to reproduce**
1. Install Formie plugin.
2. Create a new Form Template.
3. Create a new Form and use the Template in it.
4. Go to `config\general.php` file and add  `defaultTemplateExtensions` value as `['twig']`.
5. In the `Twig` template, Try to [render the form](https://verbb.io/craft-plugins/formie/docs/template-guides/rendering-forms). Ex: `{{ craft.formie.renderForm('contactForm') }}`

### Form settings
- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.5.13
- Craft version: 3.7.27.2
- Multi-site: No

**Comment by engram-design** [comment]
Hadn't actually occurred to me that this wouldn't work, but you're right that this behaviour does seem to happen. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.14",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.15](https://github.com/verbb/formie/releases/tag/1.5.15)

**Comment by anita-chouhan** [comment]
@engram-design Thank you for making the effort and providing a fix.
After upgrading to `1.5.15`, forms can now be rendered without the need for HTML in `defaultTemplateExtensions.`
